### PR TITLE
no-commit: test comment review

### PR DESCRIPTION
**Pull request by BoweiZhang-at**

### CODE REVIEW

**Review by psyclaudeZ — COMMENTED** [review]
Comment review

**Review by psyclaudeZ — COMMENTED** [review]
Another comment review

**Comment by psyclaudeZ** [comment]
This is not a comment review right?

**Review by psyclaudeZ — COMMENTED** [review]
Another comment review

**Review by psyclaudeZ — CHANGES_REQUESTED** [review]
Objection!

**Review by psyclaudeZ — APPROVED** [review]
Approved!

**Comment by BoweiZhang-at** [comment]
A normal comment shouldn't remove the tag

File: .github/workflows/state-management2.yml [hunk 1]
```diff
@@ -5,12 +5,17 @@ on:
     types: [submitted]
 
 jobs:
-  open-tag:
+  full-context:
```
Comment (BoweiZhang-at) [inline .github/workflows/state-management2.yml:8]: So does an inline comment

Comment (BoweiZhang-at) [reply .github/workflows/state-management2.yml:8]: wait what

**Comment by psyclaudeZ** [comment]
A normal comment shouldn't remove the tag

**Review by BoweiZhang-at — COMMENTED** [review]
Eh... I can comment review my own stuff...?

**Review by BoweiZhang-at — COMMENTED** [review]
Eh... I can comment review my own stuff...?

**Pull request merged**